diff --git a/interface/reweight_interface.py b/interface/reweight_interface.py
--- a/madgraph/interface/reweight_interface.py
+++ b/madgraph/interface/reweight_interface.py
@@ -1716,4 +1716,4 @@
         self.options = {'curr_dir': os.path.realpath(os.getcwd()),
-                        'rwgt_name': None}
+                        'rwgt_name': self.options['rwgt_name'] if (hasattr(self, 'options') and 'rwgt_name' in self.options) else None}
         if keep_name:
             self.options['rwgt_name'] = obj['rwgt_name']
