diff --git a/madgraph/iolibs/template_files/matrix_standalone_splitOrders_v4.inc b/madgraph/iolibs/template_files/matrix_standalone_splitOrders_v4.inc
--- a/madgraph/iolibs/template_files/matrix_standalone_splitOrders_v4.inc
+++ b/madgraph/iolibs/template_files/matrix_standalone_splitOrders_v4.inc
@@ -116,8 +116,9 @@
 	  COMMON/%(proc_prefix)sHELUSERCHOICE/USERHEL
       
 	  INTEGER POLARIZATIONS(0:NEXTERNAL,0:5)
+      COMMON/%(proc_prefix)sBORN_BEAM_POL/POLARIZATIONS
       DATA ((POLARIZATIONS(I,J),I=0,NEXTERNAL),J=0,5)/NPOLENTRIES*-1/
-      COMMON/%(proc_prefix)sBORN_BEAM_POL/POLARIZATIONS
+
 C
 C     FUNCTIONS
 C

diff --git a/madgraph/iolibs/template_files/matrix_standalone_v4.inc b/madgraph/iolibs/template_files/matrix_standalone_v4.inc
--- a/madgraph/iolibs/template_files/matrix_standalone_v4.inc
+++ b/madgraph/iolibs/template_files/matrix_standalone_v4.inc
@@ -88,8 +88,9 @@
 %(den_factor_line)s
 
 	  INTEGER POLARIZATIONS(0:NEXTERNAL,0:5)
+      COMMON/%(proc_prefix)sBORN_BEAM_POL/POLARIZATIONS
       DATA ((POLARIZATIONS(I,J),I=0,NEXTERNAL),J=0,5)/NPOLENTRIES*-1/
-      COMMON/%(proc_prefix)sBORN_BEAM_POL/POLARIZATIONS
+
 C
 C     FUNCTIONS
 C
