diff --git a/madgraph/interface/common_run_interface.py b/madgraph/interface/common_run_interface.py
--- a/madgraph/interface/common_run_interface.py
--- b/madgraph/interface/common_run_interface.py
@@ -3137,3 +3137,3 @@
             if mass and width/mass < 1e-12:
-                logger.error('The width of particle %s is too small for an s-channel resonance (%s). If you have this particle in an s-channel, this is likely to create numerical instabilities .', param.lhacode[0], width)
+                logger.warning('The width of particle %s is too small for an s-channel resonance (%s). If you have this particle in an s-channel and do not decay it without Madspin, this is likely to create numerical instabilities .', param.lhacode[0], width)
                 if CommonRunCmd.sleep_for_error:
